Makefile

```diff
@@ -30,6 +30,9 @@ docs: .paasta/bin/activate
 test: .paasta/bin/activate
 	.paasta/bin/tox -i $(PIP_INDEX_URL)
 
+api: .paasta/bin/activate
+	python3.6 ./example_cluster/setup_fake_config.py
+
 .paasta/bin/activate: requirements.txt requirements-dev.txt
```

Comment:
Although I don't really know why we have this in the first place. This is what tox "does" ?

Reply:
which tox? tox in .paasta/bin/activate?

Reply:
You can discard this comment. If we wanted to, we could use tox to create this virtualenv (or just use the tox's py36 venv) instead of manually calling pip here.